### PR TITLE
Review dplyr + include in site

### PR DESCRIPTION
**Pull request by peterdesmet**
This PR makes the dplyr tutorial part of the website. I did not review it in depth, but made some stylistic changes here and there. See the commits.

## TODO (ideally after merging)

1. Don't use the short form for grouping (currently shown as an alternative and used in the exercise outputs):

```r
# don't teach
data %>%
  count(column)

# but just
data %>%
  group_by(column) %>% # makes it clear you are grouping
  count() # easily replaceable by summarize
```

2. Every exercise block now has a `exercise = TRUE` option. Is it possible to empty those blocks when rendering the site?

### CODE REVIEW

File: src/dplyr.Rmd [hunk 1]
```diff
@@ -265,199 +266,223 @@ df %<>% f1(...) %>% f2(...)
 
 We apply all steps on the right of `%<>%` to `df` and overwrite it with the final ouput.
 
-```{r}
+```{r double_pipes}
+library(magrittr)
```
Comment (damianooldoni) [inline src/dplyr.Rmd:270]: ah, ok. You want to stress the fact that the `%<>%` comes from `magrittr`. Nice!

File: src/dplyr.Rmd [hunk 2]
```diff
@@ -265,199 +266,223 @@ df %<>% f1(...) %>% f2(...)
...
 ```
 
 ## Exercises
 
-1. Count records per sex
+1. Count records per sex:
 
-```{r}
+```{r count_per_sex, exercise = TRUE}
 data %>% count(sex)
```
Comment (damianooldoni) [inline src/dplyr.Rmd:400]: I agree perfectly with your PR comment at #11. I teach the `group_by` technique but then I show that `count` doesn't need it. It is definitely not the best approach to teach `group_by`! I will change it in a new branch after merging this one.

File: src/dplyr.Rmd [hunk 3]
```diff
@@ -265,199 +266,223 @@ df %<>% f1(...) %>% f2(...)
...
+
 head(weight_per_species_sex_gather, n = 10)
+
 head(weight_per_species_sex, n = 10)
+
 colnames(weight_per_species_sex_gather) == colnames(weight_per_species_sex)
 ```
 
-# rename and mutate
+# Rename, add and update columns
```
Comment (damianooldoni) [inline src/dplyr.Rmd:475]: I like the new titles: they give the goal of what the functions do! Thanks for noticing it.

File: src/dplyr.Rmd [hunk 4]
```diff
@@ -565,24 +596,21 @@ OT | Ottoicus dissitus Drake, 1960
...
+  species == "DO" ~ "Dondice occidentalis (Engel, 1925)",
+  species == "DS" ~ "Distonemurus desertus Krivokhatsky, 1992",
+  species == "OL" ~ "Olios scepticus Chamberlin, 1924",
+  species == "PE" ~ "Petrochelidon nigricans (Vieillot, 1817)",
+  species == "PF" ~ "Piffliella eduardi Hammer, 1979",
+  species == "OT" ~ "Ottoicus dissitus Drake, 1960",
+  species == "OX" ~ "Oxybelus pallidus Arnold, 1927",
+  is.na(species) ~ "Not Available"
+  ))
 ```
```
Comment (damianooldoni) [inline src/dplyr.Rmd:611]: I didn't check the weight of this species. So, i am quite sure their weight don't follow the values in column `weight` (or `weight_in_g`). Do you think it is a problem? I don't. It is just an example, after all and it's not worth the hassle, isn't it?

**Review by damianooldoni — APPROVED** [review]
I agree with changes. I see that you import all packages from `tidyverse` at once in `dwc-mapping.Rmd`. I am thinking to do the same in `dplyr.Rmd` but I am still not 100% sure about it. First of all, because we say it is part of a workshop which goes about `dplyr` package. Together with changes asked in #11, I will add (in a new branch) a link to the dplyr-cheatsheet from RStudio internetsite in the introduction. Thanks for checking and improving the quality of the workshop documents!

**Comment by damianooldoni** [comment]
@stijnvanhoey: I await your ok and then after merging this PR, I will start a new branch to implement the few last (final?) changes as asked by @peterdesmet in the PR #11 and very few little things as I wrote in my review, essentially adding a link to the last version of the dplyr cheatsheet, mentioning that dpyr is part of the tidyverse package bundle, thus showing the possibility to use the command `library(tidyverse)`.

**Comment by stijnvanhoey** [comment]
merging is ok to me!

**Pull request merged**